device/mockup/tt_mockup_device.hpp

```diff
@@ -21,7 +21,7 @@ class tt_MockupDevice : public tt_device {
     virtual ~tt_MockupDevice() {}
```

Comment:
Can we add building mockup device to CI? How do you build it? Can you maybe add a mockup test to tests/api?

Reply:
added very simple test.